ktor-shared/ktor-serialization/ktor-serialization-kotlinx/common/src/io/ktor/shared/serialization/kotlinx/KotlinxSerializationBase.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.shared.serialization.kotlinx
```

Comment:
`shared` in prefix is not needed anymore after #2665 
please also remove not only change package name, but also folder, which was not changed in mentioned PR